common/utils_test.go

```diff
@@ -35,3 +39,48 @@ func TestMakeDir(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, "drwx------", fileStats.Mode().String()) // only user has access
 }
+
+func TestFileExists(t *testing.T) {
```

Comment:
Just increasing the coverage in `utils.go`.  I had modified this function in a recent PR, but it had no coverage.